Sources/Components/Viewings/ViewingCell.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol ViewingCellDelegate: AnyObject {
+    func viewingCellDidSelectAddToCalendarButton(_ cell: ViewingCell)
+}
+
+class ViewingCell: UITableViewCell {
+    weak var delegate: ViewingCellDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var viewingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.spacing = ViewingCell.contentSpacing
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private lazy var dateStackView: UIStackView = {
+        let stackView = UIStackView(frame: .zero)
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private lazy var monthLabel: Label = {
+        let label = Label(style: .detail, withAutoLayout: true)
+        label.textColor = .textCritical
+        return label
+    }()
+
+    private lazy var dayTimeStackView: UIStackView = {
+        let stackView = UIStackView(frame: .zero)
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.spacing = .verySmallSpacing
+        return stackView
+    }()
+
+    private lazy var dateLabel = Label(style: .title3, withAutoLayout: true)
+
+    private lazy var dayLabel = Label(style: .body, withAutoLayout: true)
+
+    private lazy var timeLabel: Label = {
+        let label = Label(style: .detail, withAutoLayout: true)
+        label.textColor = .textSecondary
+        label.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+        return label
+    }()
+
+    private lazy var addToCalendarButton: Button = {
+        let button = Button(style: .utility, size: .small, withAutoLayout: true)
+        button.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        button.addTarget(self, action: #selector(addToCalendarButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    static let dateViewWidth: CGFloat = 56.0
+    static let cellHeight: CGFloat = 64.0
+    static let contentSpacing: CGFloat = .smallSpacing
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    // MARK: - Setup
+
+    private func setup() {
+        addSubview(viewingStackView)
```

Comment:
it's a good practice when working with `UITableViewCell`/`UICollectionViewCell` to add child views into the cell's `contentView` instead of directly (because of other features of cells, like accessory views)